### DISCUSSION TITLE
[V3] How can I create a ExampleAction in a separate class to reuse it in diferent places?

### DISCUSSION THREAD

**Discussion by dkstudio86** [filamentphp/filament#8659] [General]
Is there some documentation about this? I want to create for example a app/Filament/Actions/ExampleAction.php. A then use it in:

$table->......->actions([
  ExampleAction::make(Model $record) 
])

Thanks en advance
(and sorry for my english)

**Comment by wychoong** [comment]
https://github.com/filamentphp/filament/blob/3.x/packages/actions/src/EditAction.php
